geo/src/algorithm/rotate.rs

```diff
@@ -400,9 +410,23 @@ mod test {
     }
 
     #[test]
-    fn test_rotate_multipolygon_errors_gracefully() {
-        // an multipolygon whose centroid cannot be found should return itself, rather
-        // than panicing (e.g. an empty multipolygon, or a multipolygon composed of empty polygons)
+    fn test_rotate_empty_geometries_error_gracefully() {
```

Comment:
If I should also test some other degenerate geometries, I can add another test case or two.

Reply:
Thanks for the tests!